contentcuration/contentcuration/models.py

```diff
@@ -24,6 +31,11 @@ class Channel(ChannelMetadata):
     deleted =  models.ForeignKey('TopicTree', null=True, blank=True, related_name='deleted')
     clipboard =  models.ForeignKey('TopicTree', null=True, blank=True, related_name='clipboard')
     draft =  models.ForeignKey('TopicTree', null=True, blank=True, related_name='draft')
+    bookmarked_by = models.ManyToManyField(
+        'auth.User',
+        related_name='bookmarked_channels',
+        verbose_name=_("bookmarded by"),
```

Comment:
(also, may not be necessary -- I think this will be the default verbose name here)